### ISSUE TITLE
Samsungnotes on Mac?

### ISSUE THREAD

**Issue by qitianya** [kellwinr/galaxybook_mask#38]
Sir, your poject is great, I can access the note on win pc. However, currently I work on Mac to keep together with my roommates, well, I cannot use the samsung notes again😂 Actually, I asked the LLMs and Googled lots of web---it's a unsloved problem, though I already copied my notes to apple's Notes.
Well,  do you have any idea?

**Comment by kellwinr** [comment]
I also use a Mac right now but I switched to using OneNote already. Is Samsung Notes not running fine in VM?

**Comment by qitianya** [comment]
Yes, actully we can't find samsung notes on Mac. So I also have switched to Onenote, but it's turely a bullshit i must to say.
That's a reddit talk about sam-notes on Mac,
https://www.reddit.com/r/GalaxyTab/comments/p626rl/samsung_notes_for_mac/

**Comment by kellwinr** [comment]
The open/view in OneNote feature is also quite janky on Samsung Notes so I use OneNote on all my devices now and ditched Samsung Notes completely. Plus, Samsung Notes has never been supported on macOS, sadly there is really nothing we can do about this though.

**Comment by qitianya** [comment]
so sad
